web_ui/frontend/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "src",
-  "version": "0.1.0",
+  "version": "7.2.0",
```

```suggestion
  "version": "7.5.0",
```

If we want a version to start with, we'd better start with the latest version to release?

---

The action is set to do it automatically, unfortunately the main branch is fated to have a package.json that is forever behind.